src/transformers/processing_utils.py

```diff
@@ -796,38 +796,40 @@ class MyProcessingKwargs(ProcessingKwargs, CommonKwargs, TextKwargs, ImagesKwarg
                 if modality_key in tokenizer_init_kwargs:
                     default_kwargs[modality][modality_key] = tokenizer_init_kwargs[modality_key]
         # now defaults kwargs are updated with the tokenizers defaults.
-        # pass defaults to output dictionary
         output_kwargs.update(default_kwargs)
 
+        # gather common kwargs and remove them from individual kwargs if present
```

Comment:
Hmm, this is weird because BLIP-2 doesn't have the processor testMixin in the latest `main` branch and thus should not run the failing tests. 

In any case we shouldn't be adding hacks in code to pass the test that shouldn't be running the first place, let's better skip the failing tests in Blip-2 test file with `@unittest.skip("BLIP-2 yet doesn't have standardized API for processor kwargs")`